homework_create_more_models/README.md

```diff
@@ -22,23 +22,23 @@ class Comment(models.Model):
         return self.text
 ```
 
-You can go back to **Django models** chapter in tutorial if you need to remind yourself what each of field types means.
+You can go back to the **Django models** chapter in the tutorial if you need a refresher on what each of the field types mean.
 
-In this chapter we have new type of field:
+In this tutorial extension we have a new type of field:
 - `models.BooleanField` - this is true/false field.
 
-And `related_name` option in `models.ForeignKey` allow us to have access to comments from post model.
+The `related_name` option in `models.ForeignKey` allows us to have access to comments from within the Post model.
 
 ## Create tables for models in your database
 
-Now it's time to add our comment model to database. To do this we have to let know Django that we made changes in our model. Type `python manage.py makemigrations blog`. Just like this:
+Now it's time to add our comment model to the database. To do this we have to tell Django that we made changes to our model. Type `python manage.py makemigrations blog` in your Terminal. You should see output like this:
```

Comment:
This should be "command line" or lower case "terminal" rather than capitalized "Terminal" which is a Mac application and not applicable to all users.